File: Labos-04/src/main/java/hr/tvz/hadzalic/rentacarapp/entity/Vozilo.java
package hr.tvz.hadzalic.rentacarapp.entity;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.time.LocalDate;

@Data
@AllArgsConstructor
public class Vozilo {
    private Long id;
    private String registration;
    private String vin;
    private Integer maxNumberOfPassenger;
    private String shifter;
    private String airConditioning;
    private Integer numberOfDoors;
    private String fuelType;
    private LocalDate lastServiceDate;
    private LocalDate nextSeviceDate;
    private Double mileage;
}

File: Labos-10/frontend/src/app/interfaces/vozilo.ts
export interface Vozilo {
    id: number;
    registration: string;
    vin: string;
    maxNumberOfPassenger: number;
    shifter: string;
    airConditioning: string;
    numberOfDoors: number;
    fuelType: string;
    lastServiceDate: Date;
    nextServiceDate: Date;
    mileage: number;
}
